flask/boxwise_flask/graph_ql/type_defs.py

```diff
@@ -3,11 +3,132 @@
 
 type_defs = gql(
     """
-    type Base {
-        id: Int
+    type Box{
+        ID: ID!
+        boxLabelNumber: Int!
```

Comment:
@aerinsol what field of the Peewee model do `ID` and `boxLabelNumber` refer to? Currently the `stock` table has the columns `id` and `box_id`.

Reply:
@pylipp I think the boxLabelNumber refer to box_id and ID is refer to id is stock table. Hope this helps!

Reply:
Thanks! Then here `boxLabelNumber` needs to be defined as String.

Reply:
@pylipp yes, I think so; this should be change to String to be sync with current data model.

Reply:
@vahidbazzaz @pylipp if that's so then that's super confusing and definitely needs to be changed :D

I've updated the schema so `boxLabelNumber` is now `boxLabelIdentifier ` and it's a string. In real terms, this is the human-readable "ID" that gets written on the box. @pylipp can we change the Peewee model to something similar like `box_label_identifier` to match?

Reply:
@aerinsol I agree with you; also keeping the GraphQL schema in sync between UI and backend is of utmost importance as we use schema-first GraphQL engine.

Reply:
Got it! As a consequence, the related query should then be `getBoxDetails(boxLabelIdentifier: String)`, right?